### PR TITLE
Add backfill section to README

### PR DESCRIPTION
**Pull request by relud**
I thought this would be bigger, but it isn't, so it felt appropriate to embed it in the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,27 +6,57 @@ BigQuery ETL
...
+    investigated in [#66](https://github.com/mozilla/bigquery-etl/issues/66)
+- Should split backfilling into queries that finish in minutes not hours
+  - May output to multiple tables that are combined using copy-append
+    operations
+- May generate incremental tables from scratch
+  - Tooling is being added to automate this process in
+    [#65](https://github.com/mozilla/bigquery-etl/issues/65)
+  - May use `UPDATE` instead of equivalent `SELECT` queries to improve speed
+    - In one case adding a new column calculated from existing columns with
+      `DATE_DIFF` using `UPDATE` was twice as fast using `SELECT`
```
Comment (jklukas) [inline README.md]:
```suggestion
      `DATE_DIFF` using `UPDATE` was twice as fast as using `SELECT`
```

File: README.md [hunk 2]
```diff
@@ -6,27 +6,57 @@ BigQuery ETL
...
+
+### Backfills
+
+- Should be avoided on large datasets
+  - Backfills may double storage cost for a dataset for 90 days by moving
+    data from long-term storage to short-term storage
+    - For example regenerating `clients_last_seen_v1` from scratch would cost
+      about $1600 for the query and about $6800 for data moved to short-term
+      storage
+  - Should combine multiple backfills happening around the same time
```
Comment (jklukas) [inline README.md]: Maybe we should have a process for accumulating pending changes that would be convenient to include in the next backfill. Maybe one would PR non-urgent changes to a `.pending.sql` file.

Not in scope here, but something that popped into my head while reading.

Comment (relud) [reply README.md]: I was thinking that we could update queries to generate new columns and exclude old columns, and use views to hide changes, and then whenever we decide to backfill we would only have to run it up to the most recent change.

File: README.md [hunk 3]
```diff
@@ -6,27 +6,57 @@ BigQuery ETL
...
+  - Should delay column deletes until the next other backfill
+    - Should use `NULL` for new data and `EXCEPT` to exclude from views until
+      dropped
+- Should use copy operations in append mode to change column order
+  - Copy operations do not allow changing partitioning, changing clustering, or
+    column deletes, but using the Storage API to do these things for less than
+    on-demand query pricing, $5 per TiB as of 2019-04-01, is being investigated
+    in [#66](https://github.com/mozilla/bigquery-etl/issues/66)
```
Comment (relud) [inline README.md]: @jklukas r? on this updated wording

**Pull request merged**